docs/refarch/boundedContexts/platformConfigAndSecretMan/index.md

```diff
@@ -0,0 +1,54 @@
+# Platform Configuration And Secret Management BC
+
+{overview}
+
+## Terms
+
+Terms with specific and commonly accepted meaning within the Bounded Context in which they are used.
+
+| Term | Description |
+|---|---|
+| Term1 | Description1 |
+
+## Use Cases
+
+### Configuration & Secret Bootstrapping
+
+#### Description
+
+#### Flow Diagram
+
+![Use Case - Configuration & Secret Bootstrapping](./assets/Config&Secrets_configuration-Secrets-BootStrap_20211028.png)
+> Configuration & Secret Bootstrapping
+
+### Perform Transfer (universal mode)
+
+#### Description
+
+#### Flow Diagram
+
+![Use Case - Example REPLACE ME](./assets/useCaseExample.png)
+> example image - replace
```

Comment:
Should this be replaced as part of this PR? Couple of occurrences in this file.

Reply:
This file is part of the template sample that we have used to try and introduce a semblance of uniformity to the look and feel of the document.  I will be doing some tweaks in the coming days to the look and feel of the document and either replacing or hiding the "blank" templates wherever I find them.  Thanks :-)